docs/TutorialReact.md

```diff
@@ -271,6 +273,36 @@ test('CheckboxWithLabel changes the text after click', () => {
 
 The code for this example is available at [examples/enzyme](https://github.com/facebook/jest/tree/master/examples/enzyme).
 
+#### react-testing-library
```

Comment:
I'd personally prefer this to be above enzyme. @rickhanlonii thoughts?

Reply:
Sounds good to me as long as the intro paragraph (line 211) lists them in the same order they're outlined in, and if this is first then line 280 can't refer to "the test above"